server/prerender.js

```diff
@@ -1,18 +1,26 @@
 var prerenderio = Npm.require('prerender-node');
 var token;
 var serviceUrl;
+var protocol;
 var settings = Meteor.settings.PrerenderIO;
 
 // token
 token = process.env.PRERENDERIO_TOKEN || (settings && settings.token);
 
+
 // service url (support `prerenderServiceUrl` (for historical reasons) and `serviceUrl`)
 serviceUrl = settings && (settings.prerenderServiceUrl || settings.serviceUrl);
 serviceUrl = process.env.PRERENDERIO_SERVICE_URL || serviceUrl;
 
+
+if(protocol) {
+  prerenderio.set('prerenderProtocol', protocol);
+}
+
 if (token) {
   if (serviceUrl) prerenderio.set('prerenderServiceUrl', serviceUrl);
   prerenderio.set('prerenderToken', token);
+  prerenderio.set('prerenderProtocol', protocol);
```

Comment:
This line should have the `if (protocol)` check, just like in the line above (`if (serviceUrl) prerenderio.set('prerenderServiceUrl', serviceUrl);`).